timer.r

```diff
@@ -0,0 +1,43 @@
+# timer.r
+getwd()
+library(tictoc)
+
+# Source the three methods
+source("scripts/method1.r")
+source("scripts/method2.r")
+source("scripts/method3.r")
+
+# Time Method 1
+tic()
+source("scripts/method1.r", chdir = TRUE)
+toc()
+
+# Time Method 2
+tic()
+source("scripts/method2.r", chdir = TRUE)
+toc()
+
+# Time Method 3
+tic()
+source("scripts/method3.r", chdir = TRUE)
+toc()
+
+
+### Results
```

Comment:
Good comments. You could also have included a summary table with the times it took for each method.